go.mod

```diff
@@ -0,0 +1,23 @@
+module github.com/gruntwork-io/terragrunt
+
+go 1.12
+
+require (
+	github.com/aws/aws-sdk-go v1.16.27
+	github.com/bgentry/go-netrc v0.0.0-20140422174119-9fd32a8b3d3d
+	github.com/davecgh/go-spew v0.0.0-20161028175848-04cdfd42973b
+	github.com/go-errors/errors v0.0.0-20161205223245-8fa88b06e597
+	github.com/hashicorp/go-cleanhttp v0.0.0-20170211013415-3573b8b52aa7
+	github.com/hashicorp/go-getter v0.0.0-20171007181130-2f449c791e6a
+	github.com/hashicorp/go-version v0.0.0-20170914154128-fc61389e27c7
+	github.com/hashicorp/hcl v0.0.0-20171017181929-23c074d0eceb
+	github.com/jmespath/go-jmespath v0.0.0-20180206201540-c2b33e8439af
+	github.com/mattn/go-zglob v0.0.0-20170920030352-4ecb59231939
+	github.com/mitchellh/go-homedir v0.0.0-20161203194507-b8bc1bf76747
+	github.com/mitchellh/go-testing-interface v0.0.0-20171004221916-a61a99592b77
+	github.com/mitchellh/mapstructure v0.0.0-20171017171808-06020f85339e
+	github.com/pmezard/go-difflib v0.0.0-20151028094244-d8ed2627bdf0
+	github.com/stretchr/testify v0.0.0-20171018052257-2aa2c176b9da
+	github.com/ulikunitz/xz v0.5.4
+	github.com/urfave/cli v0.0.0-20171014202726-7bc6a0acffa5
+)
```

Comment:
What are these `.mod` / `.sum` files?

Reply:
The files allows go to compile terragrunt outside GOPATH.
https://github.com/golang/go/wiki/Modules

Reply:
Ah, OK. Could we do this in a separate PR? It requires more thinking (i.e., do we really want _both_ Go dep and Go mod, if we remove Go dep, does that break existing users, etc).

Reply:
Sure no problem. Although I didn't have any issues compiling terragrunt either inside GOPATH or outside using go.mod. GO is smart enough to handle this.